### PR TITLE
1.5 Cleanup

### PR DESCRIPTION
**Pull request by scottdixon**
- CI, Deps, Coverage.

### CODE REVIEW

**Comment by jlipps** [comment]
oohhh chrome hackage. nice.

also the console.logs can probably be removed if they were just for your own debugging.

**Comment by scottdixon** [comment]
hah! You caught me mid-hack @jlipps 
Will be sure to remove the logs.

**Comment by scottdixon** [comment]
@imurchie

**Comment by scottdixon** [comment]
Check out line 20 of `.travis.yml`

The initial NPM install misses some `appium-gulp-plugins` dependencies - happens locally too, have you seen this before?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  # Setting language to objective-c sends us to an OSX VM
+  - objective-c
+before_install:
+  # TavisCI's OSX environment gives us node 0.10.32
+  # Use NVM to update it to 0.12
+  - rm -rf ~/.nvm
+  - git clone https://github.com/creationix/nvm.git ~/.nvm
+  - source ~/.nvm/nvm.sh
+  - nvm install 0.12
+  # Chrome Driver needs Chrome >= 43.0.2357.0
```
Comment (imurchie) [inline .travis.yml]: You might note ChromeDriver version XXX needs this Chrome. This is a moving target.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  # Update to the latest version
+  - wget -P /tmp/ https://dl.google.com/chrome/mac/stable/GGRO/googlechrome.dmg
+  - hdiutil attach /tmp/googlechrome.dmg
+  - sudo cp -r /Volumes/Google\ Chrome/Google\ Chrome.app /Applications/
+before_script:
+  - rm -rf node_modules
+  - npm install
+  - cd node_modules/appium-gulp-plugins
+  - npm install
+  - cd ../..
```
Comment (imurchie) [inline .travis.yml]: Can I introduce you to `pushd` and `popd`? 

https://en.wikipedia.org/wiki/Pushd_and_popd

**Comment by imurchie** [comment]
I saw that a while back but haven't recently. We should look into it, but your solution here seems good.

File: .travis.yml (hunk 2 above)
Comment (scottdixon) [inline .travis.yml]: Nice tip! Updated

File: package.json [hunk 3]
```diff
@@ -30,13 +30,13 @@
     "appium-logger": "^2.0.0",
     "appium-support": "^2.0.0",
     "asyncbox": "^2.0.2",
-    "babel-runtime": "=5.8.24",
+    "babel-runtime": "=5.8.25",
```
Comment (imurchie) [inline package.json]: Let's keep this at 5.8.24 until we can go through all the modules. Sometimes mixing babel versions causes problems.

Comment (scottdixon) [inline package.json]: Done!

**Pull request merged**